SrfGen/srfinfo2vm.py

```diff
@@ -631,25 +632,26 @@ def plot_vm(vm_params, srf_corners, mag, ptemp):
         )
 
 
-def create_vm(args, srf_meta):
+def create_vm_gen_params(temp_dir, srf_meta, space_land=5.0, space_srf=15.0, dt=0.005, hh=0.4, pgv=5.0):
```

Comment:
since now there is two default values now with this change (one in arg one here), its probably a good idea to have a global defined value/variable to avoid potential oversights.